tests/test_numpymemmapextractor.py

```diff
@@ -107,49 +107,6 @@ def test_roundtrip(self, dtype, num_channels, num_rows, num_columns, buffer_size
         )
         check_imaging_equal(imaging_extractor1=extractor, imaging_extractor2=roundtrip_extractor)
 
-    def test_get_frames_indexing_with_single_channel(self):
```

Comment:
Do you know what's going on with these tests?

Reply:
Cline removing them because there is a mention of channel.

Reply:
Ok, after giving it a think, I suppose we don't really need to replace these tests with anything.